### PR TITLE
Feat:tooltip z index

### PR DESCRIPTION
**Pull request by aeschi**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208054401992732

### CODE REVIEW

**Comment by julizet** [comment]
looks good imho :)

File: src/components/radar/contact-points/contact-points.tsx [hunk 1]
```diff
@@ -26,23 +27,21 @@ export function ContactPoints() {
 			{contactCircles.map(
 				({ x, y, title, contactIndex, matchesSelectedTags }) => (
 					<div
+						className="group absolute cursor-pointer rounded-full bg-white flex justify-center"
```
Comment (raphael-arce) [inline src/components/radar/contact-points/contact-points.tsx:30]: question/suggestion: should we remove the `cursor-pointer` class when a contactCircle does not match selected tags?

**Review by raphael-arce — APPROVED** [review]
One question, otherwise lgtm :) 👍

File: src/components/radar/contact-points/contact-points.tsx (hunk 1 above)
Comment (aeschi) [reply src/components/radar/contact-points/contact-points.tsx:30]: good question, @julizet just said it's ok to still click on them and open the dialog, so I would keep the pointer for now.

Comment (raphael-arce) [reply src/components/radar/contact-points/contact-points.tsx:30]: makes sense. lets keep it as is then 👍

**Pull request merged**